swap/src/protocol/alice.rs

```diff
@@ -308,26 +308,42 @@ impl Behaviour {
         &mut self,
         channel: ResponseChannel<AliceToBob>,
         swap_response: SwapResponse,
-    ) {
-        self.amounts.send(channel, swap_response);
+    ) -> Result<()> {
+        let _ = self.amounts.send(channel, swap_response)?;
```

Comment:
I suspect the return value of `send` is `Result<(), _>`? Using `_` for `()` is a bit pointless :)

Reply:
There is a log statement after amounts.send() which I will have to get rid of if I get rid of the pointless binding. I didn't want to make that change in an upgrade PR.

Reply:
That is not what I meant. What I meant is keep the `?` and just remove the binding:

```suggestion
        self.amounts.send(channel, swap_response)?;
```